Makefile

```diff
@@ -407,15 +405,8 @@ else
 endif
 
 install: $(PKGCFGF) $(ARCHIVE) $(LIBRARY)
-	mkdir -p $(DESTDIR)/$(LIBDIR)
-ifeq ($(CAPSTONE_SHARED),yes)
-	$(INSTALL_LIB) $(LIBRARY) $(DESTDIR)/$(LIBDIR)
-ifneq ($(VERSION_EXT),)
-	cd $(DESTDIR)/$(LIBDIR) && \
-	rm -f lib$(LIBNAME).$(EXT) && \
-	ln -s lib$(LIBNAME).$(VERSION_EXT) lib$(LIBNAME).$(EXT)
-endif
-endif
+	mkdir -p $(LIBDIR)
```

Comment:
why did you change `mkdir -p $(DESTDIR)/$(LIBDIR)` to `mkdir -p $(LIBDIR)` here?